assets/js/translator.js

```diff
@@ -66,16 +66,17 @@ if(modeEnabled('translation')) {
         });
 
         $('.srcLang').click(function () {
-            curSrcLang = $(this).attr('data-code');
-            $('.srcLang').removeClass('active');
-            $(this).addClass('active');
-            populateTranslationList();
-            refreshLangList(true);
-            muteLanguages();
-            localizeInterface();
-            translateText();
-
-            autoSelectDstLang();
+            if(this.id !== 'detect') {
```

Comment:
If it's literally all of it, then we should just use a different selector, no? `.srcLang:not(#detect)`?